src/Features/LanguageServer/Protocol/Handler/PullHandlers/PreviousPullResult.cs

```diff
@@ -6,5 +6,5 @@
 
 namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 {
-    internal record struct PreviousPullResult(string PreviousResultId, TextDocumentIdentifier TextDocument);
+    internal record struct PreviousPullResult(string PreviousResultId, TextDocumentIdentifier TextDocument, string? DiagnosticSourceIdentifier);
```

Comment:
Is this needed?  The handler name / resultId should contain the source identifer already - so we should already be getting a separate resultId cache per source.  Can't figure out otherwise where this is getting used

Reply:
Thanks, reverted!